.github/CODEOWNERS

```diff
@@ -62,6 +62,8 @@
 /libs/shared/connected/src/lib/generalPetition/GeneralPetitionLists                           @island-is/kosmos-kaos
 /libs/api/domains/communications/                                                             @island-is/kosmos-kaos
 /apps/services/notifications/                                                                 @island-is/kosmos-kaos
+/apps/air-discount-scheme/                                                                    @island-is/kosmos-kaos @island-is/core
```

Comment:
What's the reason for having core there as well?

Reply:
Will update the login / authentication system, the idea was that the Core would be able to review it best.